crates/sui-bridge/src/types.rs

```diff
@@ -249,60 +365,13 @@ impl BridgeAction {
         bytes.extend_from_slice(BRIDGE_MESSAGE_PREFIX);
         match self {
             BridgeAction::SuiToEthBridgeAction(a) => {
```

Comment:
Here is it possible to do something like
```suggestion
            BridgeAction::SuiToEthBridgeAction(a) | BridgeAction::EthToSuiBridgeAction(a) |  BridgeAction::BlocklistCommitteeAction(a) => { 
                bytes.extend_from_slice(&a.to_bytes());
            }
```

Reply:
it does not, they need to be the same type.